snooty/semanticparser.py

```diff
@@ -226,7 +226,8 @@ def build_toctree(self) -> Dict[str, SerializableType]:
 
         # Build the toctree
         root: Dict[str, SerializableType] = {
-            "title": self.project_config.name,
+            "title": self.project_config.title,
+            "name": self.project_config.name,
```

Comment:
We actually don't need `name` in order to render the toctree, so you can remove this line. But good thinking!